components/ws-manager-bridge/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "0.1.5",
   "license": "UNLICENSED",
   "scripts": {
-    "start": "node ./dist/src/index.js",
+    "start": "node ./dist/index.js",
```

Comment:
I don't know why and/or how this worked previously 🤷

Reply:
Does not start for me.

Reply:
Fix pushed.